R/runShiny.R

```diff
@@ -0,0 +1,15 @@
+#' A wrapper function to run Shiny Apps from \code{CiteSource}.
+#' 
+#' Running this function will launch the CiteSource shiny
+#' @return CiteSource shiny app
+#' @param app CiteSource 
+#' @importFrom magrittr `%>%`
```

Comment:
This needs to be `#' @importFrom magrittr %>%` - i.e. no backticks around the pipe (https://community.rstudio.com/t/import-from-magrittr-package/4313/5)